xbmc/video/Bookmark.h

```diff
@@ -38,20 +38,24 @@ class CBookmark
    */
   bool IsPartWay() const;
 
+  bool IsMediaBookmark() const;
+  static bool IsMediaBookmark(int t);
+
   double timeInSeconds;
   double totalTimeInSeconds;
   long partNumber;
   std::string thumbNailImage;
   std::string playerState;
   std::string player;
-  long seasonNumber;
-  long episodeNumber;
+  std::string title;
 
   enum EType
   {
     STANDARD = 0,
     RESUME = 1,
-    EPISODE = 2
+    EPISODE = 2,
```

Comment:
Any way this could simply be called `MEDIA` so that it doesn't have to be extended in case there are more media types in the future?